.github/workflows/bench.yml

```diff
@@ -0,0 +1,12 @@
+name: Benchmarks
+on: [push, pull_request]
+jobs:
+  bench:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Benchmarks
+        run: npm --prefix bench start
```

Comment:
Should be `FORCE_COLOR= npm --prefix bench start` to make sure everyone is actually doing any work.

Reply:
Oh thanks, that explains the weird results 😅 It looks better now https://github.com/pioug/colorette/runs/3576204006?check_suite_focus=true#step:4:15

```
colorette × 502,111 ops/sec
chalk × 342,441 ops/sec
kleur × 372,220 ops/sec
colors × 170,611 ops/sec
ansi-colors × 213,404 ops/sec
```